episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+- Many more cases are likely to be **symptomatic** (i.e., with fever, cough, or myalgia) but might not require hospitalisation. These cases will have been identified through links to international travel to high-risk areas and through contact-tracing of contacts of confirmed cases. They might also be identified through population surveillance of, for example, influenza-like illness. 
+- The bottom part of the pyramid represents **mild** (and possibly **asymptomatic**) cases. These cases might be identified through contact tracing and subsequently via serological testing.
+
+::::::::::::::
+
+To calculate the naive CFR, the `{cfr}` package requires a data frame with three columns named:
+
+- `date`
+- `cases`
+- `deaths`
+
+:::::::::::::::::: callout
+
+### We need daily data
+
+These should give the **daily** number of reported cases and deaths. 
+
+We emphasise **daily** because this needs to show all the observations for all the days in the time frame of your analysis, even if the number of observations is zero or there are missing values.
+
+Also, this currently only works for **daily** data, not weekly data.
+
+<!-- suggest ways to deal with raw input weekly data -->
+<!-- https://github.com/epiverse-trace/cfr/issues/117 -->
+
+::::::::::::::::::
+
+Let's explore the `ebola1976` dataset, which comes from the first Ebola outbreak in Zaire in 1976, as analysed by Camacho et al. (2014).
+
+```{r}
+# Load the Ebola 1976 data provided with the package
+data("ebola1976")
+
+# view top data
+ebola1976 %>% as_tibble()
+```
+
+When we use `cfr_static()` to `data` only, we calculate the naive CFR:
+
+```{r}
+# Calculate the static CFR without adjusting for delays
+cfr_static(data = ebola1976)
+```
+
+:::::::::::::::::::::::::::::::::::::::: challenge
+
+Load the `ebola_kikwit_1995` from the `{outbreaks}` package and estimate the naive CFR.
+
+:::::::::::::::::::: hint
+
+Inspect the format of the data input.
+
+- Does it contain daily data?
+- Does the column names are as required by `cfr_static()`?
+- How would you rename column names from a data frame?
+
+::::::::::::::::::::
+
+:::::::::::::::::::: solution
+
+We can use `dplyr::rename()` to adapt the external data to fit the data input for `cfr_static()`.
+
+```{r}
+library(outbreaks)
+
+outbreaks::ebola_kikwit_1995 %>%
+  glimpse()
+
+outbreaks::ebola_kikwit_1995 %>%
+  dplyr::rename(
+    cases = onset,
+    deaths = death
+  ) %>%
+  cfr::cfr_static()
+```
+
+::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::
+
+## Biases that affect CFR estimation
+
+[Lipsitch et al., 2015](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0003846) describe two potential biases that can affect the estimation of CFR (and its possible solutions):
```

Comment:
since the episode has prerequisite delay distributions, we expect to to take too much space from this for that topic. I added a link to the glossary at https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/3a0fa129ce114f1068abc83612d92bec7dbcdf9f

For a clear dependency, we can redirect to early task episodes where we cover them with more detail

Reply:
also added reference of it in the prereq and link to early task tutorials in the glossary https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/467aeb1fd2dd37c17a50ec44313065f152704c16